Data-tranformations/fairgenomes/experiment.yarrrml.yaml

```diff
@@ -5,18 +5,36 @@ prefixes:
   EDAM: https://bioportal.bioontology.org/ontologies/EDAM?p=classes&conceptid=
   FIX: http://purl.obolibrary.org/obo/FIX_
   GENEPIO: http://purl.obolibrary.org/obo/GENEPIO_
+  fgs: https://w3id.org/fair-genomes/ontology/ 
 
 
-mappings:
+mappings: # mapping for sequencing
   fairgenomes:
     sources:
       - [experiment.csv~csv] # I am going to name it like this 
-    s: ega.dataset:$(accession_id)
+    s: ega.runs:$(accession_id) # I guess that experiment is included in ega.runs otherwise I cannot think something more appropriate from the list
```

Comment:
I am talking about this list: 
  ega.analyses: https://metadata.ega-archive.org/analyses/
  ega.dacs: https://metadata.ega-archive.org/dacs/
  ega.datasets: https://metadata.ega-archive.org/datasets/
  ega.runs: https://metadata.ega-archive.org/runs/
  ega.samples: https://metadata.ega-archive.org/samples/
  ega.studies: https://metadata.ega-archive.org/studies/